tests/functional/test_main.py

```diff
@@ -1,7 +1,10 @@
-import commands
+from future import standard_library
```

Comment:
The `future` package should be added to `requirements-dev.txt` if you want to use it in the tests.